src/i18n.ts

```diff
@@ -27,6 +27,7 @@ const i18next = i18n
           loadSuggestions: 'Suggesties laden...',
           noSuggestions: 'Geen suggesties gevonden.',
           searchPlaceholder: 'Voer een zoekterm in',
+          showAll: 'toon alles',
```

Comment:
De rest is met hoofdletters... 
```suggestion
          showAll: 'Toon alles',
```

Reply:
![image](https://user-images.githubusercontent.com/4056615/100356587-b8dd4780-2ff3-11eb-9a98-c5a147873856.png)
dat is bewust, omdat de tekst in de balk wordt gebruikt.